go.mod

```diff
@@ -4,7 +4,7 @@ go 1.12
 
 require (
 	github.com/fatih/structs v1.1.0
-	github.com/golang-jwt/jwt v3.2.1+incompatible
+	github.com/form3tech-oss/jwt-go v3.2.1+incompatible
```

Comment:
This repository is archived. According to the readme from `dgrijalva/jwt-go` the correct maintenance of this repository was moved to `github.com/golang-jwt/jwt` so the current import is correct.

Reply:
github.com/andygrunwald/go-jira@v1.13.0 requires\n\tgithub.com/dgrijalva/jwt-go@v3.2.0+incompatible: missing go.sum entry; to add it:\n\tgo mod download github.com/dgrijalva/jwt-go\n

Reply:
I am getting this error @benjivesterby